config.json

```diff
@@ -218,6 +218,15 @@
      ]
   }, {
      "difficulty": 1,
+     "slug": "triangle",
+     "topics": [
+		"control-flow (if-else statements)",
+		"Booleans",
+		"Structs",
+		"Mathematics"
+	       ]
```

Comment:
The indentation is off here with respect to the rest of the file. Can you correct this before merge?